src/workflow_components/composite_action.py

```diff
@@ -72,7 +76,7 @@ def from_dict(obj_dict) -> "CompositeActionStep":
             uses_string_obj = UsesString.analyze(uses_string=s.uses)
             if uses_string_obj.type == UsesStringType.ACTION:
                 obj = get_or_create_composite_action(
```

Comment:
This won't work at the moment.
Need to fix that somehow.
Currently relations are based on action paths. (neglecting tags)